app/controllers/identities_controller.rb

```diff
@@ -20,6 +20,17 @@ def show
     render plain: "404 Not Found", status: :not_found unless @identity
   end
 
+  def toggle_portal
+    return redirect_to root_path unless admin?
+
+    form_user
```

Comment:
this is calling form_user, but not doing anything with the return value. Do we need to call form_user here?

Reply:
we later user return value from `form_iser` in actual `if` statement for Flipper toggle. this was broken down to please rubocop, which resulted in a bit convoluted implementation from my side. i will tidy this up now! Cheers